src/WinUI/ArcGISRuntime.WinUI.Viewer/Samples/Symbology/CustomDictionaryStyle/readme.md

```diff
@@ -10,19 +10,22 @@ When symbolizing geoelements in your map, you may need to convey several pieces
 
 ## How to use the sample
 
-Pan and zoom the map to see the symbology from the custom dictionary style.
+Use the radio buttons to toggle between the dictionary symbols from the web style and style file. Pan and zoom around the map to see the symbology from the chosen dictionary symbol style. The web style and style file are slightly different to each other to give a visual indication of the switch between the two.
 
 ## How it works
 
-1. Create a new `DictionarySymbolStyle` by passing in the path to the custom style file (.stylx).
-2. Create a new `DictionaryRenderer`, providing the dictionary symbol style.
-3. Apply the dictionary renderer to a feature layer using `Renderer` property of the layer.
+1. Create a new `FeatureLayer` and append it to the Map's operational layers.
+2. Use `DictionarySymbolStyle.CreateFromFileAsync()` to create a `DictionarySymbolStyle` from a .stylx file.
+3. Create a `DictionarySymbolStyle` with a `PortalItem`.
+4. Create and apply the desired `DictionaryRenderer` to the `FeatureLayer` for each `DictionarySymbolStyle` by selecting the appropriate radio button.
 
 ## Relevant API
 
 * DictionaryRenderer
 * DictionarySymbolStyle
 * DictionarySymbolStyleConfiguration
+* Portal
```

Comment:
I don't think `Portal` is necessary here. The PortalItem is used but the Portal is only used as an input to the item. I think the PortalItem could have been created without a Portal object.